packages/core/metrics/core-metrics-collectors-server-mocks/BUILD.bazel

```diff
@@ -0,0 +1,102 @@
+load("@npm//@bazel/typescript:index.bzl", "ts_config")
+load("@build_bazel_rules_nodejs//:index.bzl", "js_library")
+load("//src/dev/bazel:index.bzl", "jsts_transpiler", "pkg_npm", "pkg_npm_types", "ts_project")
+
+PKG_DIRNAME = "core-metrics-collectors-server-mocks"
+PKG_REQUIRE_NAME = "@kbn/core-metrics-collectors-server-mocks"
+
+SOURCE_FILES = glob(
+  [
+    "src/**/*.ts",
+  ],
+  exclude = [
+    "**/*.test.*",
+    "**/*.stories.*",
+  ],
+)
+
+SRCS = SOURCE_FILES
+
+filegroup(
+  name = "srcs",
+  srcs = SRCS,
+)
+
+NPM_MODULE_EXTRA_FILES = [
+  "package.json",
+]
+
+RUNTIME_DEPS = [
+  "//packages/core/metrics/core-metrics-server",
+  "@npm//moment",
```

Comment:
NIT: for consistency with our other packages, move the `@npm` dependencies before the `//packages` ones.